### PR TITLE
Update UI, add Appbar

### PR DESCRIPTION
**Pull request by vlinh128**
## Scope of work

 - [ ] W Market (Backend)
 - [x] Frontend 
 - [ ] Smartcontract

## Types of changes

 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [x] Refactoring (no functional/api changes)
 - [ ] Build-related change (change to the build process or configuration, please attach a PR to `ops/argo-apps` here if applicable)
 - [ ] Documentation change (adding or updating project documentation)
 - [ ] Other (please describe): Add Appbar, refactor UI

### CODE REVIEW

**Comment by vlinh128** [comment]
Homepage
Screen Shot 2021-11-26 at 11 00 38
>

File: frontend/components/Appbar/index.tsx [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+                <p>Explore</p>
+            </MenuItem>
+            <MenuItem onClick={() => { router.push('/create-nft-form') }}>
+                <p>Create</p>
+            </MenuItem>
+            <MenuItem onClick={() => { router.push('/profile') }}>
+                <p>Profile</p>
+            </MenuItem>
+        </Menu>
+    );
```
Comment (m00nb0w) [inline frontend/components/Appbar/index.tsx:59]: these responsive props for mobile is not needed for now right anh Linh ?

File: frontend/components/Layout/index.tsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { Components } from "@mui/material"
+import { Box } from "@mui/system"
+import { NextPage } from "next"
+import PrimarySearchAppBar from "../Appbar"
+
+
+
```
Comment (m00nb0w) [inline frontend/components/Layout/index.tsx:7]:
```suggestion
```

File: frontend/components/nfts_items/created_nfts.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import type { GetStaticProps, NextPage } from 'next'
```
Comment (m00nb0w) [inline frontend/components/nfts_items/created_nfts.tsx:1]: should rename this file to `frontend/components/NFTItem/CreatedItems.tsx` to follow our convention and prop name

File: frontend/components/Appbar/index.tsx (hunk 1 above)
Comment (vlinh128) [reply frontend/components/Appbar/index.tsx:59]: This is menu , which wil be showed when window's size verry small.

Screen Shot 2021-11-26 at 11 14 35

**Review by m00nb0w — APPROVED** [review]

**Pull request merged**